Servers/UI/OJS.Servers.Ui/ClientApp/src/layout/header/PageHeader.tsx

```diff
@@ -1,33 +1,31 @@
 import React, { useCallback } from 'react';
-import { Link } from 'react-router-dom';
-import { ReactNotifications } from 'react-notifications-component';
 import logo from './softuni-logo-horizontal.svg';
 import PageNav from '../nav/PageNav';
 import Heading from '../../components/guidelines/headings/Heading';
 import styles from './PageHeader.module.scss';
 import { useAuth } from '../../hooks/use-auth';
 import { administrationContestsGridUrl } from '../../utils/urls';
+import { LinkButton } from '../../components/guidelines/buttons/Button';
 
 const PageHeader = () => {
     const { user } = useAuth();
 
     const renderLinks = useCallback(() => {
         const administrationLink = user.permissions.canAccessAdministration
-            ? <a href={administrationContestsGridUrl} className={styles.headerLink}>Administration</a>
```

Comment:
I see no reason for the new type `a` here. What is wrong with the current `Link`?

Reply:
The `a` is removed here.